src/Keboola/StorageApi/S3Uploader.php

```diff
@@ -193,6 +193,9 @@ private function multipartUploaderFactory(
             'Key' => $key,
             'ACL' => $acl,
             'concurrency' => $concurrency,
+            'before_upload' => function (\Aws\Command $command) {
```

Comment:
Asi tu neni up-to-date code style, mel by vyzadovat import trid (`use \Aws\Command`)

Reply:
JJ, je to tak aj pri return v tejto metode, tak som to neriesil 🤷